contracts/interchain-token-service-impl.clar

```diff
@@ -533,10 +533,11 @@
             (wrapped-source-chain (get source-chain payload-decoded))
             (minter-bytes (get minter-bytes payload-decoded))
             (minter (if (is-eq (len minter-bytes) u20) (unwrap! (principal-construct? (if (is-eq chain-id u1) 0x16 0x1a) minter-bytes) ERR-INVALID-MINTER) NULL-ADDRESS))
+            (its-hub-chain (get-its-hub-chain))
         )
-        (asserts! (not (is-eq wrapped-source-chain (get-its-hub-chain))) ERR-UNTRUSTED-CHAIN)
+        (asserts! (not (is-eq wrapped-source-chain its-hub-chain)) ERR-UNTRUSTED-CHAIN)
```

Comment:
I think this condition should be removed, since the `wrapped-source-chain` can be `axelar` if a dApp has contracts on the `axelar` chain.